docs/products/networking/nodebalancers/_index.md

```diff
@@ -1,40 +1,81 @@
 ---
 title: NodeBalancers
-description: "Linode multi-port NodeBalancers feature SSL termination, sticky sessions, passive health checks, and throttling to prevent potential abuse."
+description: "Linode's managed cloud-based load balancing service, designed to provide high availability and horizontal scaling to any application."
 tab_group_main:
     is_root: true
     title: Overview
     weight: 10
 cascade:
     date: 2020-06-02
-    product_description: "NodeBalancers are load balancers-as-a-service in the cloud, managed by Linode. They intelligently route incoming requests to backend Linodes to help your application cope with any load increase."
+    product_description: "Managed cloud-based load balancing service offering high availability and horizontal scaling to any application."
 ---
 
-## Availability
+**NodeBalancers** are managed *load balancers as a service (LBaaS)*, making load balancing accessible and easy to configure on Linode's platform. They intelligently distribute incoming requests to multiple backend Compute Instances, so that there's no single point of failure. This enables high availability, horizontal scaling, and A/B testing on any application hosted with Linode.
 
-NodeBalancers are available across [all regions](https://www.linode.com/global-infrastructure/).
+## High Availability
 
-## Pricing
+In a typical single machine configuration, issues with the machine may cause the application to stop working as expected or become inaccessible. High availability solutions remove this single point of failure through combining multiple machines (redundancy), monitoring systems, and automatic failover - all of which are implemented by NodeBalancers.
+
+## Horizontal Scaling
+
+There are two main ways to scale an application, increasing the performance and capacity to your applications. *Vertical scaling* increases or decreases the resources on the existing machines. This is achieved by [resizing](/docs/guides/resizing-a-linode/) your Compute Instances. *Horizontal scaling* adds or remove machines that are identically configured to serve your application or perform a certain task. This is commonly accomplished through a load balancing solution, like NodeBalancers. Horizontal scaling can be much more flexible and allows you to scale as needed without taking down your site while upgrading or downgrading.
```

Comment:
I think in the opening sentence, you can remove the comma and say "to increase." As it stands, it sounds like the first sentence is giving one of the ways to scale, which it does not.

Reply:
Implemented